mslib/msidp/idp.py

```diff
@@ -1092,6 +1092,9 @@ def main():
         '\033[91mWARNING: msidp is solely for development and '
         'testing purposes; do not use in production environments.\033[0m'
     )
+    if os.name == "nt":
```

Comment:
I am currently not sure if "nt" will crash because of a missing "xmlsec1" on the `import saml2.xmldsig as ds`line